tests/queries/0_stateless/03221_fixed_string_string_too_long.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE t1
```

Comment:
I don't know if there is an issue registered somewhere for which this PR adds tests. If yes, it would be nice to mention it in a comment, otherwise future readers may lack some context.

It would also be nice to add at least a few comments. I generally think that our SQL tests are severely underdocumented. For most tests, one unfortunately needs to use `git blame` and read the original PR to understand their purpose. We don't need to exaggerate commenting but at least mention
- what is tested, ideally with an issue reference
- how is it tested, e.g. describe the input and the expected output.